lib/vscode/src/vs/code/browser/workbench/workbench.ts

```diff
@@ -453,6 +493,14 @@ class WindowIndicator implements IWindowIndicator {
 	// Workspace Provider
 	const workspaceProvider = new WorkspaceProvider(workspace, payload);
 
+	// NOTE@coder: copied from upstream, modified to use our repo
+	// Home Indicator
+	const homeIndicator: IHomeIndicator = {
+		href: 'https://github.com/cdr/code-server',
+		icon: 'code',
+		title: localize('home', "Home")
+	};
```

Comment:
I swear...they add it, they remove it, they add it, they remove it 😂

Reply:
Hah, I know right? It doesn't even show up on the top though, I just added that there so that there were no compiler warnings and so that our code matched theirs.